### DISCUSSION TITLE
Two panel window

### DISCUSSION THREAD

**Discussion by cyber0pog** [iterate-ch/cyberduck#16061] [Ideas]
Two panel window, like WinSP, so that files can be quickly and easily transferred from the server to the PC

**Comment by dkocher** [comment]
You can browse to your local disk like any other connection and then copy files between open windows or tabs.

**Reply by cyber0pog** [reply]
Yes, I can, but it's very inconvenient, I'm used Total commander & WinSCP, i don't like using Explorer etc.
